### ISSUE TITLE
Variant Request: Omega Chess

### ISSUE THREAD

**Issue by yoav-rozin** [ianfab/Fairy-Stockfish#189]
Hey,
I know it probably won't be simple (because the board isn't only a square and there are 2 new pieces if i am not mistaken) but it will be great if you can add this variant.
https://www.chessvariants.com/large.dir/omega/rules.html

**Comment by ianfab** [comment]
Yes, due to the board omega chess uses it basically depends on both https://github.com/ianfab/Fairy-Stockfish/issues/53 and https://github.com/ianfab/Fairy-Stockfish/issues/38, so an implementation can only be achieved long-term, but let's see.
